### PR TITLE
rgpt: create page

### PR DESCRIPTION
**Pull request by vibovenkat123**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/rgpt.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Ask GPT3 for a more unique result using a float value between 0 and 2. (higher = more unique):
+
+`rgpt --pres {{1.2}} --i "$(git diff {{path/to/file}})"`
+
+- Ask GPT3 to review your code using a specific model:
+
+`rgpt --model {{text-davinci-003}} --i "$(git diff {{path/to/file}})"`
+
+- Make rgpt use a json output:
```
Comment (kbdharun) [inline pages/common/rgpt.md:26]:
```suggestion
- Make `rgpt` use a JSON output:
```
How about this?

File: pages/common/rgpt.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# rgpt
+
+> An automated code review tool that uses GPT3 you can use straight from your terminal.
+> More information: <https://github.com/vibovenkat123/review-gpt>.
+
+- Ask GPT3 to improve the code with no extra options:
+
+`rgpt --i "$(git diff {{path/to/file}})"`
+
+- Get a more detailed verbose output from rgpt while reviewing the code:
```
Comment (kbdharun) [inline pages/common/rgpt.md:10]:
```suggestion
- Get a more detailed verbose output from `rgpt` while reviewing the code:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by vibovenkat123** [comment]
Updated because rgpt now supports gpt3.5

**Review by blueskyson — APPROVED** [review]

**Pull request merged**